pkg/operator/minio/controller.go

```diff
@@ -90,13 +91,14 @@ func (c *MinioController) StartWatch(namespace string, stopCh chan struct{}) err
 }
 
 func (c *MinioController) makeMinioHeadlessService(name, namespace string, spec miniov1alpha1.ObjectStoreSpec, ownerRef meta_v1.OwnerReference) (*v1.Service, error) {
-	coreV1Client := c.context.Clientset.CoreV1()
-
-	svc, err := coreV1Client.Services(namespace).Create(&v1.Service{
+	svc, err := c.context.Clientset.CoreV1().Services(namespace).Create(&v1.Service{
 		ObjectMeta: meta_v1.ObjectMeta{
 			Name:      name,
 			Namespace: namespace,
-			Labels:    map[string]string{k8sutil.AppAttr: minioLabel},
+			Labels: map[string]string{
```

Comment:
What if you create a helper method to get the labels? Then you only have to define the labels in one block of code.